_docs/geyser/using-geyser-with-consoles.md

```diff
@@ -6,7 +6,7 @@ permalink: /geyser/using-geyser-with-consoles/
 
 All consoles can join third-party servers - including Geyser servers - with workarounds. Xbox One, Nintendo Switch, and PS4 systems can join third-party servers using a third-party program called BedrockConnect. For technical information about the program, including how to run your own setup, see [their GitHub repository](https://github.com/Pugmatt/BedrockConnect) (*This program is not affiliated with GeyserMC*). Other methods are also available for use.
 
-There is also a tool anyone can set up that allows you to connect to the server via adding a user to your friends list, as well as a tool admins can set up that allows connecting from a linked account of a user on the server. This tool can be found [here](https://github.com/rtm516/MCXboxBroadcast).
+There is also a tool anyone can set up that allows you to connect to the server via adding a user to your friends list which can be found [here](https://github.com/Androecia/FriendConnect), as well as a tool admins can set up that allows connecting from a linked account of a user on the server which can be found [here](https://github.com/rtm516/MCXboxBroadcast).
```

Comment:
```suggestion
There is also a tool anyone can set up that allows you to connect to the server via adding a user to your friends list - [FriendConnect](https://github.com/Androecia/FriendConnect), and [MCXboxBroadcast](https://github.com/rtm516/MCXboxBroadcast). The latter one can even be set up as a Geyser Extension.
```
Both tools work very similarly, so i'm not sure why they are differentiated here.

Reply:
I decided to keep them seperate because MCXboxBroadcast as an extension was borked at the time. Now that I see it's fixed, I approve.